src/lib/src/x86_calcbonds.cpp

```diff
@@ -36,12 +36,12 @@ void CalcBondsInner(const VectorToScalarT<VectorT> *coords0,
                     const VectorToScalarT<VectorT> *box, std::size_t n,
                     VectorToScalarT<VectorT> *out) {
   auto vecbox = BoxT(box);
-
```

Comment:
I noted that forcing the use of BigVecT gave me a like 0.4 ns, but this is problem and implementation dependent.